sentry_sdk/integrations/opentelemetry/span_processor.py

```diff
@@ -45,6 +47,38 @@ def __new__(cls):
 
         return cls.instance
 
+    def __init__(self):
+        # type: () -> None
+        @add_global_event_processor
+        def link_trace_context_to_error_event(event, hint):
+            # type: (Dict[str, Any], Dict[str, Any]) -> Dict[str, Any]
+            if hasattr(event, "type") and event["type"] == "transaction":
+                return event
+
+            otel_span = get_current_span()
+            if not otel_span:
+                return event
+
+            ctx = otel_span.get_span_context()
+            trace_id = format_trace_id(ctx.trace_id)
+            span_id = format_span_id(ctx.span_id)
+
+            invalid_span = (
+                trace_id == "00000000000000000000000000000000"
```

Comment:
Does python OTEL not have a method or at least constant for these?

Reply:
done